src/content/getting-started/observe-your-clusters-and-apps/_index.md

```diff
@@ -1,15 +1,107 @@
+description: Check cluster and app metrics with the observability tools provided with the Giant Swarm platform.
 weight: 70
-last_review_date: 2024-04-05
+last_review_date: 2024-08-10
 owner:
   - https://github.com/orgs/giantswarm/teams/sig-docs
 user_questions:
-  - How do I observe the platform metrics and logs for my application?
-  - What do I need to do to observe the platform metrics and logs for my application?
+  - How do I observe the platform metrics for my application?
+  - What do I need to do to observe the platform metrics for my application?
 ---
 
-<!--
-Show how to access Grafana and basic dashboard and logs
- -->
+In the Giant Swarm platform, every cluster benefits from automatic monitoring by default. It allows you to observe the cluster health and application behaviour simultaneously with little effort. An in-cluster agent scrapes the metrics and sends them to a central location on the management cluster, where you can interact with them through our managed Grafana instance.
+
+## Requirements
+
+First step, you need a running workload cluster. If you don't have one, please first [create a workload cluster]({{< relref "/getting-started/provision-your-first-workload-cluster" >}}). Second, you need to deploy the `hello-world` application explained [in previous step]({{< relref "/getting-started/install-an-application" >}}). Further, make sure you have installed [`jq`](https://jqlang.github.io/jq/download/) on your local.
+
+In case you are running your own application, you need to make sure that your application is already [instrumented](https://opentelemetry.io/docs/concepts/instrumentation/) to export metrics.
+
+Also, consider that ingesting new metrics into the platform impacts your costs. The central monitoring system's resource consumption is actually related to the number of metrics it has to handle, so please choose which metrics you want to ingest.
+
+## Step 1: Create a service monitor
+
+In case of running the `hello-world` application, there is a baked-in service monitor that scrapes the metrics from the application. You can enable it by setting `serviceMonitor.enabled` to `true` in the values of the helm chart. If you are running your own application, you need to create a service monitor that scrapes the metrics from your application. This is an example of a service monitor you can use:
+
+```yaml
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    application.giantswarm.io/team: team-rocket
+    app.kubernetes.io/instance: my-service
+  name: my-service
+  namespace: my-namespace
+spec:
+  endpoints:
+  - interval: 60s
+    path: /metrics
+    port: web
+  selector:
+    matchLabels:
+      app.kubernetes.io/instance: my-service
+```
+
+An important bit to notice is the `application.giantswarm.io/team` label. This label is necessary as it's required for the `Prometheus` agent to discover the target. At the same time, it's used as unique identifier for your tenant or team.
```

Comment:
```suggestion
An important bit to notice is the `application.giantswarm.io/team` label. This label is necessary as it's required for the `Prometheus` agent to discover the target.
```

This might change, it might be better to not say it?